eng/pipelines/templates/jobs/archetype-sdk-tests-jobs.yml

```diff
@@ -23,78 +23,84 @@ parameters:
 - name: Location
   type: string
   default: ''
-- name: SubscriptionConfiguration
-  type: string
-  default: $(sub-config-azure-cloud-test-resources)
 - name: ServiceDirectory
   type: string
   default: not-specified
+- name: CloudConfigurations
+  type: object
+  default:
+    PublicCloud:
```

Comment:
Can we keep the names the same as they are used everywhere else, like azure-cli, e.g. `AzureCloud`? It'll be more user-friendly.